framework/CodeInterfaces/RELAP5/RELAPparser.py

```diff
@@ -77,7 +77,7 @@ def __init__(self,inputFile,addMinorEdits=False):
         self.presentStopTrip[self.maxNumberOfDecks] = copy.deepcopy(pStop)
         pStop = None
         prevDeckLineNum = lineNum + 1
-      elif line.strip().startswith("600"):
+      elif line.strip().startswith("600 "):
```

Comment:
The white space is needed to understand if this is a "600" stop trip or just a card starting with "600" (e.g. 6000000)